source/server/admin/dynamic_stats.js

```diff
@@ -0,0 +1,88 @@
+// Before first checkin:
+//   * top-n algo to limit compute overhead with high cardinality stats with user control of N.
+//   * alternate sorting criteria, reverse-sort controls, etc.
+
+// Follow-ups:
+//   * render histograms
+//   * detect when user is not looking at page and stop or slow down pinging the server
+//   * hieararchical display
+//   * json flavor to send hierarchical names to save serialization/deserialization costs
+
+let current_stats = new Map;
+
+const param_id_prefix = 'param-1-stats-';
+
+function loadStats() {
+  val = name => name + '=' + document.getElementById(param_id_prefix + name).value;
```

Comment:
nit: maybe use some other name, `val` means `const` in some languages

Reply:
sure; I haven't cleaned this up yet. Actually that's a lint error and I will run this through an online linter before marking ready for review.